cmd/fleetctl/package.go

```diff
@@ -377,13 +377,31 @@ func packageCommand() *cli.Command {
 			}
 
 			path, _ = filepath.Abs(path)
+			pathBase := filepath.Base(path)
+			var installInstructions = "double-click the installer"
+			var deviceType string
+			switch c.String("type") {
+			case "pkg":
+				installInstructions += fmt.Sprintf(" or run the command `sudo installer -pkg \"%s\" -target /`", pathBase)
+				deviceType = "macOS"
+			case "deb":
+				installInstructions += fmt.Sprintf(" or run the command `sudo apt install \"%s\"`", pathBase)
+				deviceType = "Debian-based Linux"
+			case "rpm":
+				installInstructions += fmt.Sprintf(" or run the command `sudo dnf install \"%s\"`", pathBase)
+				deviceType = "RPM-based Linux"
```

Comment:
Q: Do double clicking `deb`/`rpm` works?

Reply:
It's always possible that something on the user's system is configured differently, but by default double clicking on an RPM or DEB should open the Software Center for that platform and give the user the option to install the package through the GUI